README.md

```diff
@@ -25,6 +25,16 @@ $ cargo install thokr
 $ docker run -it coloradocolby/thokr
 ```
 
+## Arch Linux
+
+On Arch Linux, you can install it from AUR:
```

Comment:
nit: I kinda like keeping READMEs minimal and to the point. Cargo and Docker don't have descriptions -- if someone is running arch I'm thinking this is probably not a necessary comment.

Reply:
It's ok, I'm agree